### PR TITLE
statistics: add table.GetRowCountByColumnRanges method.

### PR DESCRIPTION
**Pull request by winoros**
Forget to add this method in the previous pr.

PTAL @hanfei1991 @shenli

### CODE REVIEW

File: statistics/table.go [hunk 1]
```diff
@@ -217,6 +217,15 @@ func (t *Table) GetRowCountByIntColumnRanges(sc *variable.StatementContext, colI
 	return c.getIntColumnRowCount(sc, intRanges, float64(t.Count))
 }
 
+// GetRowCountByColumnRanges estimates the row count by a slice of ColumnRange.
+func (t *Table) GetRowCountByColumnRanges(sc *variable.StatementContext, colID int64, colRanges []*types.ColumnRange) (float64, error) {
```
Comment (hanfei1991) [inline statistics/table.go]: Add test for this function

File: statistics/table.go [hunk 2]
```diff
@@ -276,35 +285,40 @@ func getPseudoRowCountByIndexRanges(sc *variable.StatementContext, indexRanges [
 	return totalCount, nil
 }
 
-func getPseudoRowCountByColumnRange(sc *variable.StatementContext, tableRowCount float64, ran types.ColumnRange) (float64, error) {
+func getPseudoRowCountByColumnRange(sc *variable.StatementContext, tableRowCount float64, columnRanges []*types.ColumnRange) (float64, error) {
```
Comment (zimulala) [inline statistics/table.go]: s/ByColumnRange/ByColumnRanges ?
Or you can add the function named  `getPseudoRowCountByColumnRanges`.

Comment (winoros) [inline statistics/table.go]: Oh, i should add `s` there.

File: statistics/table.go (hunk 1 above)
Comment (winoros) [inline statistics/table.go]: added.

**Comment by hanfei1991** [comment]
@shenli PTAL

**Pull request merged**